pkg/api/rest/resttest/resttest.go

```diff
@@ -1022,7 +950,7 @@ func (t *Tester) testWatchFields(obj runtime.Object, initWatchFn InitWatchFunc,
 	}
 }
 
-func (t *Tester) testWatchLabels(obj runtime.Object, initWatchFn InitWatchFunc, emitFn EmitFunc, labelsPass, labelsFail []labels.Set, actions []string) {
+func (t *Tester) testWatchLabels(obj runtime.Object, emitFn EmitFunc, labelsPass, labelsFail []labels.Set, actions []string) {
```

Comment:
I initially though that it would be much more difficult to change this file and changes would be much more invasive.
These changes are very uncontroversial - nice :)